packages/eslint-plugin/package.json

```diff
@@ -9,18 +9,19 @@
         "lint": "run-p lint:es",
         "lint:es": "es-lint",
         "lint-fix": "es-lint --fix",
-        "test": "mocha"
+        "test": "SWC_NODE_PROJECT=./tsconfig.test.json mocha --require @swc-node/register --watch-extensions ts,tsx 'test/**/*.{ts,tsx}'"
     },
     "dependencies": {
-        "@typescript-eslint/utils": "^5.55.0",
-        "eslint": "^8.36.0"
+        "@typescript-eslint/utils": "^6.5.0",
+        "eslint": "^8.48.0"
     },
     "devDependencies": {
         "@blueprintjs/node-build-scripts": "^8.0.1",
+        "@swc/core": "^1.3.80",
+        "@swc-node/register": "^1.6.7",
```

Comment:
this seems to work better than ts-node and is more modern. got the suggestion from https://github.com/typescript-eslint/typescript-eslint/issues/7284#issuecomment-1696101984